src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/otel-adot-log-ingestion-serverless.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: OTEL log ingestion solution for ADOT lambda layer collector
```

Comment:
I think we should explode the acronyms in docs titles, even if it leads to long titles.

So, OTEL should be OpenTelemetry and ADOT should be... whatever ADOT is :)

Also, I'm pretty sure Lambda is always capitalized.

If you need to go shorter, you might try: "OpenTelemetry logs for [ADOT]"

Where ADOT is whatever those words stand for.

Reply:
If so, another alternative might be "Logs for AWS Distro for OpenTelemetry Lambda functions"